plugins/ComposeAuth/build.gradle.kts

```
@@ -0,0 +1,91 @@
+plugins {
+    alias(libs.plugins.kotlin.multiplatform)
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.compose)
+}
+
+description = "Extends gotrue-kt with composable"
+
+repositories {
+    mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+}
+
+@OptIn(org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi::class)
+kotlin {
+    targetHierarchy.default()
+    jvm {
+        jvmToolchain(8)
+        compilations.all {
+            kotlinOptions.freeCompilerArgs = listOf(
+                "-Xjvm-default=all",  // use default methods in interfaces,
+                "-Xlambdas=indy"      // use invokedynamic lambdas instead of synthetic classes
+            )
+        }
+    }
+    androidTarget {
+        publishLibraryVariants("release", "debug")
+    }
+    js(IR) {
+        browser {
+            testTask {
+                enabled = false
+            }
+        }
+        nodejs {
+            testTask {
+                enabled = false
+            }
+        }
+    }
+    ios()
+    iosSimulatorArm64()
+
+    sourceSets {
+        all {
+            languageSettings.optIn("kotlin.RequiresOptIn")
+            languageSettings.optIn("io.github.jan.supabase.annotations.SupabaseInternal")
+            languageSettings.optIn("io.github.jan.supabase.annotations.SupabaseExperimental")
+        }
+        val commonMain by getting {
+            dependencies {
+                api(project(":"))
+                api(project(":gotrue-kt"))
+                implementation(compose.ui)
+                implementation(compose.runtime)
+            }
+        }
+        val androidMain by getting {
+            dependsOn(commonMain)
+            dependencies {
+                api(libs.android.play.store.auth)
+                implementation(libs.android.google.id)
+                implementation(libs.kotlinx.coroutines.play.services)
+                implementation(libs.androidx.activity.compose)
+            }
+        }
+        val jvmMain by getting {
+            dependsOn(commonMain)
+        }
+
+        val appleMain by getting {
+            dependsOn(commonMain)
+        }
+        val jsMain by getting {
+            dependsOn(commonMain)
+        }
+    }
+}
+
+android {
+    namespace = "io.github.jan.supabase.composeauth"
```

```suggestion
    namespace = "io.github.jan.supabase.compose.auth"
```
We should probably use the package name